### PR TITLE
Bnc 962328

### PR DESCRIPTION
**Pull request by schubi2**
AutoYaST: Reporting timemout errors only. (bnc#962328)

### CODE REVIEW

File: package/yast2-ca-management.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Jan 21 13:06:17 CET 2016 - schubi@suse.de
+
+- AutoYaST: Reporting timemout errors only. (bnc#962328)
```
Comment (imobach) [inline package/yast2-ca-management.changes]: `timemout` -> `timeout`

File: src/include/ca-management/new_cert_read_write.rb [hunk 2]
```diff
@@ -1932,7 +1933,7 @@ def create_default_CA_certificate
             _(
               "The default certificate can also be created in\nthe CA Management module.\n"
             )
-          )
+          ) unless Mode.autoinst
```
Comment (imobach) [inline src/include/ca-management/new_cert_read_write.rb]: Should it be skipped also during `autoupgrade`? In that case, maybe you would like to use `Mode.auto`. If no, just ignore this comment :)

File: src/include/ca-management/new_cert_read_write.rb [hunk 3]
```diff
@@ -2013,7 +2014,7 @@ def create_default_CA_certificate
           _(
             "The default certificate can also be created in\nthe CA Management module.\n"
           )
-        )
+        ) unless Mode.autoinst
```
Comment (imobach) [inline src/include/ca-management/new_cert_read_write.rb]: Same as above.

File: src/include/ca-management/popup.rb [hunk 4]
```diff
@@ -80,7 +82,8 @@ def showErrorCaManagement
       if Ops.greater_than(Builtins.size(message), 0)
         Builtins.y2error("Showing error: %1", messageMap)
         if CommandLine.StartGUI
-          if Ops.greater_than(Builtins.size(description), 0)
+          if Ops.greater_than(Builtins.size(description), 0) &&
+            !Mode.autoinst # Show description while normal installation only.(bnc#962328)
```
Comment (imobach) [inline src/include/ca-management/popup.rb]: Same as above.

File: src/include/ca-management/new_cert_read_write.rb (hunk 2 above)
Comment (schubi2) [inline src/include/ca-management/new_cert_read_write.rb]: The module will not be called by autoupgrade. But it does not hurt if we are taking both to be sure.

File: package/yast2-ca-management.changes (hunk 1 above)
Comment (schubi2) [inline package/yast2-ca-management.changes]: That's becoming a running gag.....:-)

**Comment by imobach** [comment]
LGTM

**Pull request merged**